### PR TITLE
Feat: Adds `Slider` & `PriceRange` component 

### PR DESCRIPTION
**Pull request by hellofanny**
## What's the purpose of this pull request?

- [x] Moves `Slider` component to @faststore/components
- [x] Extracts `Slider` styles from starter to @faststore/ui
- [x] Adds `Slider` doc page
- [x] Refactor `PriceRange` component and add to @faststore/components
- [x] Adds  `PriceRange` styles to @faststore/ui
- [x] Adds `PriceRange` doc page

How to test it?

run `yarn workspace site dev` and check the Slider and PriceRange components  doc page or test it using the doc preview link: 
- [Slider Component](https://faststore-site-git-feat-slider-component-fs-764-faststore.vercel.app/components/atoms/slider)
- [PriceRange Component](https://faststore-site-git-feat-slider-component-fs-764-faststore.vercel.app/components/organisms/price-range)

### Starters Deploy Preview

[Core Replaces PriceRange FSUI ](https://github.com/vtex-sites/nextjs.store/pull/348)

slider_controler_left

### CODE REVIEW

**Comment by emersonlaurentino** [comment]
why can I only skip 10 out of 10? there should be a prop that defines the fraction I can skip, don't you think?

https://user-images.githubusercontent.com/10627086/212332256-0d9c02bc-5305-45d1-af47-1b999b8abc78.mov

**Comment by emersonlaurentino** [comment]
The [Radix UI Slider](https://www.radix-ui.com/docs/primitives/components/slider) uses the `step` prop to define the interval that should be used, I think we should have something like that.

**Comment by hellofanny** [comment]
> The [Radix UI Slider](https://www.radix-ui.com/docs/primitives/components/slider) uses the `step` prop to define the interval that should be used, I think we should have something like that.

@emersonlaurentino actually the user can pass this step prop, I just left 10 as default 🙇 Should this prop be required? 

https://user-images.githubusercontent.com/3356699/212338371-be488ed2-5e6c-42fd-9e3a-a4497919e4e4.mov

<PriceRange
      max={{ absolute: 500, selected: 250 }}
      min={{ absolute: 0, selected: 100 }}
      formatter={formatter}
      step={1}
    />

File: apps/site/pages/components/organisms/price-range.mdx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import { TokenTable, TokenRow, TokenDivider } from 'site/components/Tokens'
+import { OverviewSection } from 'site/components/OverviewSection'
+import { PriceRange } from '@faststore/ui'
+import PriceRangeUsage from 'site/components/PriceRange/PriceRangeUsage'
+
+<header>
+
+# Price Range
+
+The Price Range component is a slider that allows users to select a maximum and minimum price from a range.
```
Comment (renatamottam) [inline apps/site/pages/components/organisms/price-range.mdx:18]:
```suggestion
The Price Range component is a [Slider](/components/atoms/slider) that allows users to select a maximum and minimum price from a range.
```

File: packages/components/src/organisms/PriceRange/PriceRange.tsx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+   */
+  variant?: PriceProps['variant']
+  /**
+   * Formatter function that transforms the raw price value and render the result.
+   */
+  formatter: PriceProps['formatter']
+  /**
+   * Returns the value of element's class content attribute.
+   */
+  className?: string
```
Comment (renatamottam) [inline packages/components/src/organisms/PriceRange/PriceRange.tsx:19]: Why we need this?

**Review by renatamottam — APPROVED** [review]
All looks good to me, fan!

**Review by emersonlaurentino — APPROVED** [review]

File: packages/components/src/organisms/PriceRange/PriceRange.tsx (hunk 2 above)
Comment (hellofanny) [reply packages/components/src/organisms/PriceRange/PriceRange.tsx:19]: Are we allowing the user to pass class to our components? 🤔 
I think I will remove it for now

**Review by eduardoformiga — APPROVED** [review]
Wowww 🤩 , wonderful!

**Pull request merged**